src/robot/libdocpkg/model.py

```diff
@@ -150,14 +150,10 @@ def to_dictionary(self):
         }
 
     def _convert_arguments(self):
-        return [self._arg_to_dict(a) for a in self.args]
-
-    def _arg_to_dict(self, arg_info):
-        return {
-            'name': arg_info.name,
-            'type': arg_info.type_repr,
-            'default': arg_info.default_repr,
-            'kind': arg_info.kind,
-            'required': arg_info.required,
-            'repr': unicode(arg_info)
-        }
+        return [{'name': a.name,
+                 'type': a.type_repr,
+                 'default': a.default_repr,
+                 'kind': a.kind,
+                 'required': a.required,
+                 'repr': unicode(a)
+                 } for a in self.args]
```

Comment:
In my earlier comment I meant that `_convert_arguments` could be removed and the list comprehension it contained could be just used where it was called. `_arg_to_dict` was fine as it was. Sorry for not being clear about that. Good thing is that the code is fine like this as well.